app/controllers/projects_controller.rb

```diff
@@ -100,4 +107,16 @@ def projects_for_user
   def require_project
     @project = (Project.find_by_param!(params[:id]) if params[:id])
   end
+
+  def as_csv
+    projects = Project.order(:id).all
```

Comment:
Being explicit about it, since there's a controller test that assumes it's ordered by ID.